OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -814,12 +811,12 @@ public void ResolveOrder(Actor self, Order order)
 			else if (order.OrderString == "ReturnToBase" && rearmableInfo != null && rearmableInfo.RearmActors.Any())
 			{
 				if (!order.Queued)
+				{
 					UnReserve();
+					self.CancelActivity();
```

Comment:
Why not put `order.Queued` as an argument in Line 819 like you did in Line 798?

Reply:
I just overlooked that, I think. Updated.